ivy/functional/backends/numpy/device.py

```diff
@@ -12,17 +12,31 @@
 
 
 def dev(x: np.ndarray, /, *, as_native: bool = False) -> Union[ivy.Device, str]:
-    if as_native:
-        return "cpu"
```

Comment:
I know numpy doesn't have native devices, but `NativeDevice` in numpy backend is defined as `str` so I guess it should be returning something?